streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala

```diff
@@ -176,7 +176,7 @@ class ReceiverSuite extends TestSuiteBase with Timeouts with Serializable {
       blockGenerator.addData(count)
       generatedData += count
       count += 1
-      Thread.sleep(1)
+      Thread.`yield`()
```

Comment:
PS: It most certainly will compile! From the relevant section of the Scala language specification, 2.9 (p. 5):
"
Example 1.1.2 Backquote-enclosed strings are a solution when one needs to ac- cess Java identifiers that are reserved words in Scala. For instance, the statement Thread.yield() is illegal, since yield is a reserved word in Scala. However, here’s a work-around:
Thread.‘yield‘()
"